test/spec.yml

```diff
@@ -124,6 +125,13 @@
             "bn invoke momentousgiants"
             "curl -H X-Binaris-Api-Key:* https://*"
 
+- test: Test show
+  steps:
+    - in: BINARIS_ACCOUNT_ID=1066 bn show --accountId
```

Comment:
What do you think about using a positional, for `bn show apiKey`, a bit like `git config user.name`?

Reply:
I wanted to do it initially, but then you get into weirdnesses about supporting multiple positionals.  @rylandg WDYT?

Reply:
I think multiple positional arguments is pretty easy with `yargs`. I personally am indifferent in regards to the decision.